### DISCUSSION TITLE
How would you use tailwind css with vanilla html?

### DISCUSSION THREAD

**Discussion by eshankiyer** [tailwindlabs/tailwindcss#5230] [Help]
Title

**Comment by simonswiss** [comment]
Hey!

Tailwind CSS is "regular" CSS, so you just apply classes to your HTML elements.

If your question is around how to setup a project with Tailwind CSS, you can either use the [Tailwind CLI](https://tailwindcss.com/docs/installation#using-tailwind-cli), or use a build tool like Vite, as presented in this video:

https://youtu.be/qYgogv4R8zg?t=420

**Comment by eshankiyer** [comment]
Hmmmmmm. Thanks for the help, it worked just fine! Except javascript elements don't work. I don't think Tailwind's custom JavaScript can be imported. Ill check the tutorial videos for more guidance
